2020-01-06 15:53:51,836 ERROR: Exception on /edit_profile [GET] [in c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\vuno\srv\learing.python\python\mypy\microblog\app\routes.py", line 87, in edit_profile
    form = EditProfileForm()
  File "c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\wtforms\form.py", line 212, in __call__
    return type.__call__(cls, *args, **kwargs)
TypeError: __init__() missing 1 required positional argument: 'original_username'
2020-01-06 15:54:20,282 INFO: Microblog startup [in C:\Users\vuno\srv\learing.python\python\mypy\microblog\app\__init__.py:47]
2020-01-06 15:54:22,755 ERROR: Exception on /edit_profile [GET] [in c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\vuno\srv\learing.python\python\mypy\microblog\app\routes.py", line 87, in edit_profile
    form = EditProfileForm()
  File "c:\users\vuno\srv\learing.python\python\mypy\microblog\envflask\lib\site-packages\wtforms\form.py", line 212, in __call__
    return type.__call__(cls, *args, **kwargs)
TypeError: __init__() missing 1 required positional argument: 'original_username'
